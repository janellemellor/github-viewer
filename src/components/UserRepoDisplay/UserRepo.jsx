import React from 'react';
import PropTypes from 'prop-types';

const UserRepo = ({ repoName, repoUrl }) => (
  <li>
    <p>{repoName}</p>
    <p>{repoUrl}</p>
  </li>
);

UserRepo.propTypes = {
  repoName: PropTypes.string.isRequired,
  repoUrl: PropTypes.string.isRequired 
};

export default UserRepo;

