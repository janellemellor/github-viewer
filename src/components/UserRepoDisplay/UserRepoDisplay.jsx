import React from 'react';
import PropTypes from 'prop-types';

const UserRepoDisplay = ({ repos }) => (
  <>
    <p>{repos}</p>
  </>
);

UserRepoDisplay.propTypes = {
  repos: PropTypes.string.isRequired 
};

export default UserRepoDisplay;
