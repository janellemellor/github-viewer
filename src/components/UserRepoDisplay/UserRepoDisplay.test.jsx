import React from 'react';
import { shallow } from 'enzyme';
import UserRepoDisplay from './UserRepoDisplay.jsx';

describe('UserRepoDisplay', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<UserRepoDisplay repos='https://github.com/janellemellor?tab=repositories' />);
    expect(wrapper).toMatchSnapshot();
  });
});

