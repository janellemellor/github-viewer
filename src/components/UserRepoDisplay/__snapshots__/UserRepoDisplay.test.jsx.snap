// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserRepoDisplay matches a snapshot 1`] = `
<Fragment>
  <p>
    https://github.com/janellemellor?tab=repositories
  </p>
</Fragment>
`;
